packages/govuk-frontend/src/govuk/components/character-count/character-count.mjs

```diff
@@ -145,7 +145,7 @@ export class CharacterCount extends GOVUKFrontendComponent {
       throw new ElementError({
         componentName: 'Character count',
         element: $textareaDescription,
-        identifier: `#${textareaDescriptionId}`
+        identifier: `Hint element #${textareaDescriptionId}`
```

Comment:
@claireashworth I'm not sure about the descriptor here.

In this case, the element that causes the error is the container for the count message at the bottom of a Character Count
![Screenshot 2023-10-19 at 11 44 06](https://github.com/alphagov/govuk-frontend/assets/22524634/534c2c49-c5c5-4a3a-bfcc-27c7d3f4c0fa)

I've called it "Hint element" because it's using our Hint component, but the text above the textarea ("Do not include...") is more properly the "Hint". Other options might be "Description element" or "Count message element", but wanted to get your thoughts.

Reply:
We call it "count message" (8 times) in the [**Character count** guidance](https://design-system.service.gov.uk/components/character-count/)

>The count message appears below the textarea so that:
>
>* it’s clearly separate from any hint text or error message above the textarea

But we do say "character count message" once and call it a "static message" without JavaScript:

>This component uses JavaScript. If JavaScript is not available, users will see a static message in place of the count message

Reply:
The "count message"'s have it!